src/references/defisdk/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as defiSdkAdapterRegistryABI } from './adapterRegistryABI.json';
+
+export const DEFI_SDK_ADAPTER_REGISTRY_ADDRESS =
```

Comment:
can we move this to some env?

Reply:
I defer this one to @jinchung who wrote it originally.